### DISCUSSION TITLE
Default Date for Photo's with unknown date

### DISCUSSION THREAD

**Discussion by Frazzer951** [photoprism/photoprism#1851] [Q&A] [answered]
Is there a way to change the default date for photos/videos with an unknown date? 
Currently, when I open the search tab, to quickly browser for a photo, every photo and video that doesn't have a date shows up at the top. Is there a way to make this the other way around where they would be at the bottom instead? 
I have it sorted at 'Newest First' but since they have an 'Unknown Date' they are the first things that appear.

**Comment by lastzero** [comment, marked as answer]
Yes, file system mod time.
